SCIP OptSuite 10.0.0

svigerske opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

scip-ci committed:
update hash for 10.0.0

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19608695378](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19608695378). Examine the logs at this URL for more detail.</sub>

svigerske committed:
update version numbers

svigerske committed:
papilo license file changed

svigerske committed:
fix about:summary

- its a CIP framework, including a MIP solver

svigerske committed:
pack also the new lpiexact headers

svigerske committed:
update package dependencies

- add highs for papilo and gcg
- add jansson for gcg
- remove cppad - it is redistributed with scip

svigerske committed:
update cmake build flags

- remove PARASCIP flag, which has been deprecated a while ago
- remove setting EXPRINT and SYM to their current defaults
- enable HIGHS
- no longer need CMAKE_POLICY_VERSION_MINIMUM=3.5
- enable LTO
- disable UG, as fscip is not packaged anyway
- remove unused Boost_LIB_DIAGNOSTIC_DEFINITIONS=ON
- enable OpenMP explicitly

svigerske committed:
include more license files and extend license specification

svigerske committed:
use static debug version of jansson on windows

- conda-forge::jansson does not contain a decent jansson DLL currently

svigerske committed:
disable fortran compiler on windows to save disk space

- should only disable lusol of papilo

Commit:
MNT: Re-rendered with conda-smithy 3.53.2 and conda-forge-pinning 2025.11.22.22.21.1

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict. Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the `@` notation in a comment) if you believe this is a bug.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19614563571](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19614563571). Examine the logs at this URL for more detail.</sub>
